src/core_modules/capture-core/hooks/useCurrentOrgUnitInfo.js

```diff
@@ -1,10 +1,5 @@
 // @flow
 import { useSelector } from 'react-redux';
 
-export const useCurrentOrgUnitInfo = (): {|id: string, name: string, path: string|} =>
-    useSelector((
-        {
-            organisationUnitRoots: { captureRoots: { roots } },
-            currentSelections: { orgUnitId },
-        }) => roots.find(({ id }) => id === orgUnitId))
-    || {};
```

Comment:
It looked to me this wasnt necessary so I went back on a simpler way of doing this